.php_cs_rules.dist

```diff
@@ -1,33 +0,0 @@
-<?php
-
-$config = PhpCsFixer\Config::create()
-    ->setRules([
```

Comment:
I don't really want a coding standards war in here ... :)
There is however a '@Symfony' group in php-cs as well. Maybe include that to be in line with the SF standards?

My version is a bit more advanced and uses slightly different settings.
Gonna post it here to cross-reference. Maybe we can discuss the options one to one during a call?

https://gist.github.com/veewee/d7884c3e479be4994610e5dbd56245b7